Library/Formula/sqoop.rb

```diff
@@ -34,4 +34,9 @@ def caveats; <<-EOS.undent
     for Sqoop to work.
     EOS
   end
+
+  test do
+    ENV["JAVA_HOME"] = `/usr/libexec/java_home`.chomp
```

Comment:
We can set JAVA_HOME for users in install:
`bin.env_script_all_files(libexec/"bin", Language::Java.java_home_env)`